tests/simplex/compute_point_locations_01.output

```diff
@@ -70,87 +70,140 @@ DEAL::
 DEAL::Print out results in 3D: 
 DEAL::At cell 0:
 DEAL::    qpoints 0: 
-DEAL::        reference position  : (0.00000 0.00000 0.00000)
-DEAL::        physical position   : (0.00000 0.00000 0.00000)
-DEAL::        FE mapping position : (0.00000 0.00000 0.00000)
-DEAL::    qpoints 1: 
 DEAL::        reference position  : (0.00000 0.00000 0.333333)
 DEAL::        physical position   : (0.00000 0.00000 0.333333)
 DEAL::        FE mapping position : (0.00000 0.00000 0.333333)
+DEAL::    qpoints 1: 
+DEAL::        reference position  : (0.333333 0.00000 0.666667)
+DEAL::        physical position   : (0.333333 0.00000 0.666667)
+DEAL::        FE mapping position : (0.333333 0.00000 0.666667)
```

Comment:
So the changes here seem to indicate that we now, by changing the algorithm, happen to get another cell which shares the edge with the cell that we identified before. This seems uncritical to me, but I wonder how stable this test is going to be, and if configurations with slightly different roundoff behavior will still pass the test.